File: Python developer/step06/6.01/6.1.06.py
'''

Напишите программу-дешифратор шифра цезаря

Ввод:

s – строка состоящая из букв a-z

step – шаг шифра, натуральное число

Вывод:

res – расшифрованная строка

Sample Input:

def
3

Sample Output:

abc

'''

s = input()
step = int(input())
new_letter = 0
res = ''
for letter in s:
    new_letter = (ord(letter) - step - 97) % 26 + 97
    res += chr(new_letter)
print(res)

File: JavaScript. A3 Tasks/6  Задачи по темам/6.01/6.1.06.js
/*

Даны две вещественные точки на прямой -- x1x1​ и x2x2​.

Найти расстояние между ними.

Sample Input:

2
5

Sample Output:

3

*/

x1 = Number(prompt());
x2 = Number(prompt());
if (x2>x1) {
    console.log(x2-x1);
} else {
    console.log(x1-x2);
